packages/react/src/components/FileUploader/FileUploader-test.js

```diff
@@ -6,27 +6,86 @@
  */
 
 import React from 'react';
-import { CloseFilled16 } from '@carbon/icons-react';
-import FileUploader, { FileUploaderButton, Filename } from '../FileUploader';
-import FileUploaderSkeleton from '../FileUploader/FileUploader.Skeleton';
-import { mount, shallow } from 'enzyme';
 import { settings } from 'carbon-components';
+import { Close16, CheckmarkFilled16 } from '@carbon/icons-react';
+import { mount, shallow } from 'enzyme';
+import FileUploader, { FileUploaderButton, Filename } from './FileUploader';
+import FileUploaderDropContainer from './FileUploaderDropContainer';
+import FileUploaderItem from './FileUploaderItem';
+import FileUploaderSkeleton from '../FileUploader/FileUploader.Skeleton';
+import InlineLoading from '../InlineLoading';
 
 const { prefix } = settings;
 
 describe('Filename', () => {
-  const mountWrapper = mount(<Filename name={'trees.jpg'} />);
+  describe('Renders as expected', () => {
```

Comment:
Style nit: only module names should be capitalized 

Typically, renders as expected can be refactored into snapshots like:

```js
it('should render', () => {
  const classNames = [
    `${prefix}--loading`,
    `${prefix}--file-close`,
    `${prefix}--file-complete`,
  ];
  const icons = [InlineLoading, Close16, CheckmarkFilled16];
  const statuses = ['uploading', 'edit', 'complete'];

  status.forEach((status, i) => {
    // Prefer rendering full tree
    const wrapper = mount(
      <Filename iconDescriptor="mock description" status={status} />
    );

    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find(icons[i]).length).toBe(1);
  });
});
```